pkg/models/system.go

```diff
@@ -0,0 +1,37 @@
+package models
+
+import (
+	"time"
+
+	"github.com/google/uuid"
+)
+
+// System represents something that may be tested for 508 compliance
+// TODO: if we are using the `SystemFromIntake` strategy, the "db" field tags would be unecessary;
+// but if we are trying to read the System type straight from sqlx we WOULD need the "db" field tags
+type System struct {
+	IntakeID    uuid.UUID  `json:"intakeId" db:"id"` // TODO: is this actually necessary, if LCID is really the identifier?
```

Comment:
Per the discussion earlier today, it seems that Helen believes the LCID is the actual identifier of note, and the `IntakeID` will probably not be necessary, as long as we have the other fields we need from the `SystemIntake` where the LCID was issued.